src/AddIns/Uno.UI.MediaPlayer.WebAssembly/HtmlMediaPlayer.cs

```diff
@@ -319,9 +336,28 @@ private void OnHtmlMetadataLoaded(object sender, EventArgs e)
 		{
 			Duration = NativeMethods.GetDuration(_activeElement.HtmlId);
 		}
+		PlayerState = HtmlMediaPlayerState.Opening;
 		OnMetadataLoaded?.Invoke(this, Duration);
 	}
 
+	private void OnHtmlSuspended(object sender, EventArgs e)
+	{
+		if (this.Log().IsEnabled(Uno.Foundation.Logging.LogLevel.Debug))
+		{
+			this.Log().Debug($"Media Suspended [{Source}]");
+		}
+
+		_activeElement = IsVideo ? _htmlVideo : IsAudio ? _htmlAudio : default;
+		_activeElementName = IsVideo ? "Video" : IsAudio ? "Audio" : "";
+		if (_activeElement != null)
+		{
+			PlayerState = NativeMethods.GetPaused(_activeElement.HtmlId) ? HtmlMediaPlayerState.Paused : HtmlMediaPlayerState.Playing;
+			_activeElement.SetCssStyle("visibility", "visible");
```

Comment:
Why do we need to change the visibility in this case?

Reply:
I will remove. it is overzealous, to show the image.
For once in the test Suspended came before SourceLoaded.